conference/react-js/src/containers/SignUpForm/SignUpPopUp/SignUpPopUp.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ShowMainPage from './ShowMainPage';
+
+const SignUpPopUp = props => (
+  <div className="pop-up-wrapper">
+    <div className="pop-up">
+      <h3 className="pop-up__title">
+        You&#39;ve successfully registered.
+      </h3>
+      <p className="pop-up__notification">
+        <span className="pop-up__notification-user">{props.userName}</span>
+          ,&nbsp;go to&nbsp;
```

Comment:
Do you need non-breaking space? May be will be better to use ```&ensp;``` ?

Reply:
never heard about &ensp;